crates/scarb-api/Cargo.toml

```diff
@@ -14,4 +14,5 @@ serde.workspace = true
 serde_json.workspace = true
 thiserror.workspace = true
 which.workspace = true
+semver = "1.0.0"
```

Comment:
We should add that to a top lvl Cargo.toml and inherit it here instead

Reply:
It is moved there in next pr, here we use it only here.